cf-serverd/server_transform.c

```diff
@@ -993,7 +993,7 @@ static void NewHostToOldACL(Auth *old, const char *host)
 
             /* Identify next fragment: */
             const char *d = strchr(dot, '.');
-            size_t len = d ? d - dot : strlen(dot);
+            size_t len = d ? (size_t) (d - dot) : strlen(dot);
```

Comment:
This should be type-casted implicitly on the assignment, right?

Reply:
I guess the compiler implicitly typecasts this to a `long int`, thus an explicit type cast to `size_t` is required.

`server_transform.c:996:30: error: operand of ?: changes signedness from ‘long int’ to ‘size_t’ {aka ‘long unsigned int’} due to unsignedness of other operand [-Werror=sign-compare]`